R/appveyor.R

```diff
@@ -5,25 +5,34 @@ AppVeyorCI <- R6Class(
 
   public = list(
     get_branch = function() {
-      Sys.getenv("APPVEYOR_REPO_BRANCH")
+      self$has_env("APPVEYOR_REPO_BRANCH")
     },
     get_tag = function() {
-      Sys.getenv("APPVEYOR_REPO_TAG")
+      self$has_env("APPVEYOR_REPO_TAG")
     },
     is_tag = function() {
       self$get_tag() != ""
     },
     get_slug = function() {
-      Sys.getenv("APPVEYOR_REPO_NAME")
+      self$has_env("APPVEYOR_REPO_NAME")
     },
     get_build_number = function() {
-      paste0("AppVeyor build ", Sys.getenv("APPVEYOR_BUILD_NUMBER"))
+      paste0("AppVeyor build ", self$has_env("APPVEYOR_BUILD_NUMBER"))
```

Comment:
?

Reply:
Here and in `.travis.R` something might have gone wrong. I'll clean up.